README.md

```diff
@@ -11,3 +11,9 @@ Before running `photoshell`, you will need to have a working Python 3.4
 environment (including pip and virtualenv). To run `photoshell`, just do `make
 run` from the project directory. The first time you launch `photoshell` it may
 take a few minutes to install dependencies.
+
+## Dependencies
++ Python3.4
++ Virtualenv
++ Pip
++ ImageMagick-devel
```

Comment:
This is a Fedora specific package name. On Ubuntu and Debian it's `libmagickwand-dev` and on Arch you just install `imagemagick`.